run_lint.sh

```diff
@@ -0,0 +1 @@
+./gradlew lint
```

Comment:
This will run the lint inspection locally on the project - we can probably get rid of this or add a note in the Contributing docs?